src/main/resources/templates/error.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+    <head>
+        <title>Server error</title>
+    </head>
+    <body>
+        <div layout:fragment="content">
+
+            <div class="text">
+                <h1 class="heading-xlarge">Sorry we are experiencing technical difficulties</h1>
```

Comment:
Can the class be updated here to `govuk-heading-l` for use with the most up to date stylesheet please